Assets/Scripts/UI/Menu's/MenuController.cs

```diff
@@ -1,93 +1,11 @@
 using UnityEngine;
 using UnityEngine.SceneManagement;
 
-public sealed class MenuController : MonoBehaviour
+public abstract class MenuController : MonoBehaviour
 {
-    [Header("Main menu")]
-    [SerializeField] private string mainScene;
```

Comment:
```suggestion
    [SerializeField] private string mainScene = "MainMenu";
```
Ik zal deze gewoon terug halen, dan is die functie beneden "GoToMainMenu" niet hard coded. Maar je kan wel een default value mee kunnen geven, zo hoe je niet langs elk script gaan en aanpassen.